_pages/algorithm-development/advanced-algorithm-development/local-development.md

```diff
@@ -9,6 +9,7 @@ show_related: true
 author: jon_peck
 image:
     teaser: /post_images/local_development/local_development.png
+robots: noindex/nofollow
```

Comment:
The syntax [according to Google](https://developers.google.com/search/reference/robots_meta_tag) is specified as comma-separated values, so these would be `noindex, nofollow`. Preferably, these can also be combined into [a single directive](https://developers.google.com/search/reference/robots_meta_tag#directives_1) of `none` to achieve the same goal:

```md
robots: none
```

Reply:
Ahh you're right. Thanks for the heads up. Should be updated now.